README.md

```diff
@@ -16,20 +16,29 @@ If you have a WooCommerce.com account, you can [start a chat or open a ticket on
 ### Logging
 The plugin offers logging that can help debug various problems. You can enable debug mode in the main plugin settings panel under the `Enable debug mode` section.
 By default plugin omits headers in the requests to make the logs more readable. If debugging with headers is necessary you can enable the headers in the logs by setting `wc_facebook_request_headers_in_debug_log` option to true.
-## Development 
+## Development
 ### Developing
 - Clone this repository into the `wp-content/plugins/` folder your WooCommerce development environment.
-- Install dependencies: 
+- Install dependencies:
 	- `npm install`
 	- `composer install`
 - Build assets:
 	- `npm start` to build a development version
 - Linting:
 	- `npm run lint:php` to run PHPCS linter on all PHP files
+- Testing:
+	- `./bin/install-wp-tests.sh <test-db-name> <db-user> <db-password> [db-host]` to set up testing environment
+	- `npm run test:php` to run PHP unit tests on all PHP files
 
 #### Production build
 
 - `npm run build` : Builds a production version.
 
 ### Releasing
-Refer to the [wiki for details of how to build and release the plugin](https://github.com/woocommerce/facebook-for-woocommerce/wiki/Build-&-Release). 
+Refer to the [wiki for details of how to build and release the plugin](https://github.com/woocommerce/facebook-for-woocommerce/wiki/Build-&-Release).
+
+### PHPCS Linting and PHP 8.1+
+
+We currently do not support PHPCS on PHP 8.1+ versions. Please run PHPCS checks on PHP 8.0 or lower versions.
+
+Alternately, you can run PHPCS checks on PHP8.1+ versions by appending `?? ''` code within `trim()` at Line 280 of file /vendor/wp-coding-standards/wpcs/WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php and Line 194 of file /vendor/wp-coding-standards/wpcs/WordPress/Sniffs/WP/I18nSniff.php
```

Comment:
I'm leaning towards not putting the code and line details here in the readme, because it can get outdated quite fast and easily, e.g. when someone modify the files, then the line number mentioned here may not be accurate anymore.

Alternatively, we can include a link to our discussion comment in this PR, so that people can get more context details if they want to.

If you do make the changes here, do also put an "end of line" at the end of the file, even though there is no warning symbol in GitHub. 😄

Reply:
>I'm leaning towards not putting the code and line details here in the readme, because it can get outdated quite fast and easily, e.g. when someone modify the files, then the line number mentioned here may not be accurate anymore.

This makes sense. Also, we can't assume that others will see exact same errors as us in their local setup.

>Alternatively, we can include a link to our discussion comment in this PR, so that people can get more context details if they want to.

Sounds good. Folks should be able to make the desired changes and make things work with PHP8.1+ if they read through this PR.